src/layer/__tests__/fixtures.auth.ts

```diff
@@ -1,4 +1,4 @@
-import { S2L2ALayer, BBox, CRS_EPSG4326 } from 'src';
+import { S2L2ALayer, BBox, CRS_EPSG4326 } from '../../index';
 import { MimeTypes } from '../const';
```

Comment:
Since `MimeTypes` is exported in `src/index.ts`, could it be in the import above?

Reply:
Changed

Reply:
Resolving